vllm/model_executor/models/blip.py

```diff
@@ -20,6 +20,11 @@
 from vllm.multimodal.utils import (cached_get_tokenizer,
                                    repeat_and_pad_placeholder_tokens)
 from vllm.sequence import VLLM_TOKEN_ID_ARRAY_TYPE, SequenceData
+from vllm.utils import is_cpu
+
+USE_XFORMERS_OPS = not is_cpu()
+if USE_XFORMERS_OPS:
+    from xformers import ops as xops
```

Comment:
Perhaps it would be better to try to import `xformers` directly and set it to `None` if unavailable?